js/interface.js

```diff
@@ -2802,6 +2807,20 @@ $('form').validator({
         return true;
       }
 
+      return false;
+    },
+    'validation-authentication-key': function($el) {
+      var authKey = $el.val().replace(/\\n/g, '\n');
```

Comment:
I don't feel this is necessary.

If the user has entered `\n` in the textarea, we don't want to allow it. They need to be made aware that it's a mistake.